docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -79,11 +79,11 @@ application, follow these steps:
    Try http://[::1]:3000/ping
    ```
 
-2. In a browser, navigate to [http://[::1]:3000](http://127.0.0.1:3000) or
+1. In a browser, navigate to [http://[::1]:3000](http://127.0.0.1:3000) or
```

Comment:
- In the "view file", the number is correct, but maybe we should use the right numbering? 
- another nitpick: do we want to point user directly to API Explorer URL http://127.0.0.1:3000/explorer, instead of go to the root and click explorer.  No strong opinion, just think this might be simpler.

Reply:
> In the "view file", the number is correct, but maybe we should use the right numbering? 

I noticed that for `numbered lists` in MD format, you can specify `1.` every time 
so that is automatically renumbers for you. See https://www.markdownguide.org/basic-syntax/#ordered-lists .  

Otherwise, we have to keep the numbering in proper order ourselves. Making them all `1.` saves us the hassle.

Reply:
> another nitpick: do we want to point user directly to API Explorer URL http://127.0.0.1:3000/explorer, instead of go to the root and click explorer. No strong opinion, just think this might be simpler.

I guess I wanted them to see this screen:

![image](https://user-images.githubusercontent.com/6864736/66515798-bcd79e80-eaad-11e9-9ca5-b47232834a42.png)


also, when they execute : `npm start` is outputs 

![image](https://user-images.githubusercontent.com/6864736/66515869-e1337b00-eaad-11e9-8140-6d686c58f83c.png)

which tells them the server is starting at localhost:3000 ; and some users expect a home page for the server at that address.

I guess I wanted to show them there is a home screen, and they have two links to choose from.

Reply:
^ @dhmlau